<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetPanel</class>
 <widget class="QWidget" name="DatasetPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dataset Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Histogram Color by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colorByCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="legendListWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sort Output</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addSortField">
          <property name="toolTip">
           <string>Add Field</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeSortField">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Remove selected field</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveSortField">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Move selected field up</string>
          </property>
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="sortByListWidget"/>
      </item>
      <item>
       <widget class="QGroupBox" name="limitCheckBox">
        <property name="title">
         <string>Limit Output</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="limitSpinBox">
           <property name="showGroupSeparator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>20000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="limRandomButton">
           <property name="text">
            <string>Random Subset</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="limTopButton">
           <property name="text">
            <string>Top</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
